src/kernel/src/main.rs

```diff
@@ -174,8 +174,16 @@ fn mount_pfs<G: AsRef<Path>>(fs: &Fs, game: G) -> bool {
     // Open PFS image.
     let mut path = game.as_ref().to_path_buf();
 
-    path.push("uroot");
-    path.push("pfs_image.dat");
+    #[cfg(unix)]
+    {
+        path.push("uroot");
+        path.push("pfs_image.dat");
+    }
+    #[cfg(windows)]
+    {
+        path.push("\\uroot");
+        path.push("\\pfs_image.dat");
```

Comment:
This does not look right. `PathBuf` should already handled this for us.

Reply:
When @Mou-Ikkai used the parser it pushed uroot and pf_image.dat as a unix path for some reason, even though the Windows path parses correctly.

Reply:
The problem is it was working correctly before we switched the argument parser. My idea is we should write a test for the argument parser to see if it does what we expected.